app/models/document.rb

```diff
@@ -7,7 +7,7 @@ class Document
   include DateHelper
   include PublishingHelper
 
-  attr_reader :update_type
+  attr_reader :update_type, :imported
```

Comment:
Could this status be used to make sure that the licence isn't imported twice, for example if the script is run a second time? This could happen if the script fails the first time, and has to be re-run.

Reply:
That would be good to have but would need extra work implementing, as this flag would need to be persisted on the content item. We could potentially use the `content_id` or other unique identifier to work out if the document has been imported before (require an extra request to Publishing API) and skip if so, however we'd also need to consider the scenario where the content has changed, in which case we'd either need to update the new content on the existing licence or unpublish the old one first. 

Definitely something to consider but I think this is out of scope for this piece of work.